Make it easier to import project in Eclipse using Team Project Set

dandar3 opened the pull request:
* README.md - how to steps
* projectSet.psf - Use HTTPS URLs instead of SSH (failing with 'Auth fail' error for SSH URLs)

kingargyle commented:
@dandar3 can you open an eclipse bug for this, and then update the Commit message to link this PR to the bug.   Once that is done, I can merge the request.

dandar3 commented:
thank you @kingargyle, will do!

dandar3 commented:
@kingargyle 
Is it just my local or the project fails to build with:

`[ERROR] Failed to execute goal on project org.eclipse.andmore.swt: Could not resolve dependencies for project org.eclipse.andmore:org.eclipse.andmore.swt:eclipse-plugin:0.5.2-SNAPSHOT: The following artifacts could not be resolved: com.android.tools.build:builder:jar:2.3.3, com.android.tools:sdklib:jar:25.3.3, com.android.tools.ddms:ddmlib:jar:25.3.3, com.android.tools:dvlib:jar:25.3.3, com.android.tools:repository:jar:25.3.3, com.android.tools:sdk-common:jar:25.3.3: Failure to find com.android.tools.build:builder:jar:2.3.3 in https://nexus.arcsmed.at/content/repositories/homer.core/ was cached in the local repository, resolution will not be reattempted until the update interval of homer-core has elapsed or updates are forced -> [Help 1]`

v2.3.3 seem to be missing from Maven Central:
https://repo.maven.apache.org/maven2/com/android/tools/build/builder/

Adding JCenter repository to main pom.xml seems to solve the problem. 
Any thoughts?

kingargyle commented:
Tagging @andrew-bowley as he did some recent work on the deve branch.  Make sure you have synchronized your build with the latest.   I know he added some items to update Andmore to use the latest SDKs.

kingargyle commented:
@andrew-bowley please take a look at this and provide any insight you may have.

andrew-bowley commented:
I have reproduced the problem by building Andmore on a pristine Linux OS which did not have Maven installed. It turns out that the Android dependencies are downloaded from JCenter which is not natively wired into Maven. The quick fix is to add a profile to the Maven settings.xml configuration file, which is what Bintray recommends. I will attach a file with the required settings. The better fix is to add JCenter to the Andmore target file and I will organize this asap.
[maven-jcenter-config.txt](https://github.com/eclipse/andmore/files/1566953/maven-jcenter-config.txt)

Note in the attached file I use for the url "http://" instead of "https://". This is because the latter gives me "unknown host" error on my Linux OS. I do not know why this is happening, so the latter my be OK for you.

dandar3 commented:
@andrew-bowley 
Thank you for confirming, Andrew!

I've added the JCenter repository to main pom.xml on Windows with Maven 3.5.2 and it worked fine for me with both HTTPS and HTTP (HTTP would probably not be an issue since we're accessing the public part of it). The error is odd, that should indicate it is unable to resolve the name through DNS, not when unable to use the protocol. 

I agree, settings.xml would be a bit unconvenient to setup as an additional step for everyone, pom.xml should be easier.

```diff --git a/pom.xml b/pom.xml
index a8a48320..b0301205 100644
--- a/pom.xml
+++ b/pom.xml
@@ -76,6 +76,13 @@
                 <enabled>false</enabled>
             </snapshots>
 		</repository>
+		<repository>
+			<id>jcenter</id>
+			<url>https://jcenter.bintray.com/</url>
+			<snapshots>
+				<enabled>false</enabled>
+			</snapshots>
+		</repository>
 	</repositories>
 
 	<pluginRepositories>

```

hrj commented:
@dandar3 I was able to build andmore after adding the jcenter repo to `pom.xml`. The change from `ssh` to `https` would be useful for those building without a ssh key. Thanks!

dandar3 commented:
Sorry I got sidetracked, will create the ticket so we can push this further.

dandar3 committed:
[532865] Project Set easier import in Eclipse

Change GitHub URLs from SSH to HTTPS to make it easier to import the project in Eclipse

https://bugs.eclipse.org/bugs/show_bug.cgi?id=532865
Signed-off-by: Dan Dar3 <dan.dar33@gmail.com>

dandar3 commented:
@kingargyle
Finally managed to do it, please review and let me know if anything else needed, thank you!

kingargyle commented:
@dandar3 thanks, will try and get to it tonight.
